src/irmin-pack/irmin_pack.mli

```diff
@@ -63,6 +65,10 @@ module type Stores_extra = sig
   val sync : repo -> unit
   (** [sync t] syncs a readonly pack with the files on disk. Raises
       [invalid_argument] if called by a read-write pack.*)
+
+  val migrate : Irmin.config -> unit Lwt.t
+  (** [migrate conf] migrates a store with configuration [conf] to the current
+      version. Raises [RO_Not_Allowed] if called by a readonly instance. *)
```

Comment:
```suggestion
      version. *)
```

Reply:
it does raise this exception (https://github.com/mirage/irmin/pull/1015/files#diff-1beb7e2142825f197755316a39d00699R607). you think we shouldn't mention it?